core/src/main/java/org/apache/accumulo/core/security/Authorizations.java

```diff
@@ -45,6 +46,9 @@ public class Authorizations implements Iterable<byte[]>, Serializable, Authoriza
   private final Set<ByteSequence> auths;
   private final List<byte[]> authsList; // sorted order
 
+  private static final Pair<Set<ByteSequence>,List<byte[]>> EMPTY_INTERNAL_COLLECTIONS =
```

Comment:
Because this is private it seems internal is implied - do you think that EMPTY_AUTHS would still provide the same clarity, but improve readability? (just a personnel preference / suggestion)

Reply:
I think the variable name `EMPTY_AUTHS` could be confused with the `EMPTY` variable on line 55.

Reply:
`DEFAULT_EMPTY`?  To me, INTERNAL and COLLECTIONS are not adding information and the length of the name reduces readability instead of adding clarity.

Reply:
While I agree that  `INTERNAL in `EMPTY_INTERNAL_COLLECTIONS` may be redundant given that the variable is private, I don't agree that the length of the name reduces readability or that the name is confusing. `DEFAULT_EMPTY` as a variable name may contain less characters, but I don't think it provides *more* clarity.

Reply:
This is OBE, this variable was removed.